input/pagecontent/comparison.md

```diff
@@ -493,6 +632,267 @@ AU Core compliant resources are compliant with US Core requirements for Missing
 ##### Suppressed Data
 US Core does not include requirements for Suppressed Data.
 
+##### Additional Must Support elements
```

Comment:
Merge into the section Additional requirements.

Reply:
Done